static/js/statPanel.js

```diff
@@ -56,12 +70,121 @@ var startJSPanelWithChart = function( jsPanel, widthOfPanel, heightOfPanel, char
-		});
-	};
+           });
+
+           break;
+        case "accidentsPerMonthStacked":
+            var groupedAccidentsByYear = _.countBy(app.markers.pluck("created"), function(item) {
+               return item.substring(0,4);
+            });
+            var groupedAccidentsByMonth = _.countBy(app.markers.pluck("created"), function(item) {
+              return item.substring(5,7);
+            });
+            var numberOfYears = Object.keys(groupedAccidentsByYear).length;
+            var jsonAccidentsByMonth = _.map(groupedAccidentsByMonth, function(numOfAccidents, month) {return {"label": month}});
+            jsonAccidentsByMonth = _.sortBy(jsonAccidentsByMonth, 'label');
+
+            var groupedAccidentsForAllYears = _.countBy(app.markers.pluck("created"), function(item) {
+              return item.substring(0,7);
+            });
+
+            var dataPerMonth = [];
+            var dataObj = {};
+            var dataOfValues = [];
+            var dataValue = {};
+            var tempProp;
+            for (i = 0; i < numberOfYears; i++) {
+                var numOfMonthTotal = Object.keys(groupedAccidentsForAllYears).length;
+                dataOfValues = [];
+                for (j = numOfMonthTotal-1; j >= 0; j--) {
+                     if (Object.keys(groupedAccidentsForAllYears)[j].substring(0,4)===Object.keys(groupedAccidentsByYear)[i]){
+                        dataValue = new Object();
+                        dataValue = {value: groupedAccidentsForAllYears[Object.keys(groupedAccidentsForAllYears)[j]].toString()};
+                        dataOfValues.push(dataValue);
+                        tempProp = Object.keys(groupedAccidentsForAllYears)[j];
+                        delete groupedAccidentsForAllYears[tempProp];
+                     }
+                 }
+                dataObj = new Object();
+                dataObj = {seriesname: Object.keys(groupedAccidentsByYear)[i].toString(), data: dataOfValues};
+                dataPerMonth.push(dataObj);
+            }
+
+            var categories = {category: jsonAccidentsByMonth};
+
+            FusionCharts.ready(function(){
+		     var statChart = new FusionCharts({
+			   "type": "stackedcolumn2d",
+			   "renderAt": "myChart",
+			   "width": chartWidth,
+			   "height": chartHeight,
+		       "dataFormat": "json",
+		       "dataSource": {
+			   "chart": {
+				  "caption": "מספר התאונות בחודש",
+				  "xAxisName": "חודשים - מקובץ מכל השנים שנבחרו",
+				  "yAxisName": "מספר תאונות",
+				  "theme": "fint",
+                  "showSum": "1",
+				  "labelFontSize": "15",
+				  "yAxisNameFontSize": "20",
+				  "xAxisNameFontSize": "20",
+				  "captionFontSize": "25",
+			   },
+               "categories": categories,
+               "dataset": dataPerMonth
+			  }
+			});
+
+			statChart.render();
+           });
+
+           break;
+           //var groupedAccidentsByMonth = _.countBy(app.markers.pluck("created"), function(item) {
```

Comment:
The commented block is the same chart but with a regular column chart and not a stacked chart, so it shows aggregated accidents in each month fro, all of the selected years.